### PR TITLE
New themes

### LINKED ISSUE #93: Dark theme option for preview theme
When I try to set the theme for the preview window, none of the dark themes (i.e Dark Mauve/Dark Flagship Terrastruct) are listed in the picker.

### PR DESCRIPTION
**Pull request by BarryNolte**
This fixes #93, to a point.  I'm going to hold off on completely fixing it until https://github.com/terrastruct/d2/issues/831 is resolved.

### CODE REVIEW

File: src/themePicker.ts [hunk 1]
```diff
@@ -32,6 +32,9 @@ const themes: ThemeItem[] = [
   new ThemeItem("Everglade green", 104),
   new ThemeItem("Buttered toast", 105),
   new ThemeItem("Dark mauve", 200),
+  new ThemeItem("Terminal", 300),
```
Comment (gavin-ts) [inline src/themePicker.ts:35]:
```suggestion
  new ThemeItem("DarkFlagshipTerrastruct", 201),
  new ThemeItem("Terminal", 300),
```

https://github.com/terrastruct/d2/blob/66ecf134b45b32d4fc2bc37dba3b8043599d6494/d2themes/d2themescatalog/catalog.go#L10-L33

**Review by BarryNolte — COMMENTED** [review]
Don't forget to update package.json too.

File: package.json [hunk 2]
```diff
@@ -159,6 +159,9 @@
             "Earth tones",
             "Everglade green",
             "Buttered toast",
+            "Terminal",
+            "Terminal Grayscale",
+            "Origami",
```
Comment (gavin-ts) [inline package.json:164]:
```suggestion
            "Origami",
            "Dark Flagship Terrastruct",
```

but below "Dark mauve"

File: package.json [hunk 3]
```diff
@@ -159,6 +159,10 @@
...
+            "Dark Flagship Terrastruct",
```
Comment (gavin-ts) [inline package.json:165]: this should be below `Dark mauve` ideally

**Review by gavin-ts — APPROVED** [review]
what is the not completely fixed part?

**Comment by BarryNolte** [comment]
The dark-mode flag and whatever changes happen in bug #831 that could ripple through the current functionality.

**Comment by gavin-ts** [comment]
> The dark-mode flag and whatever changes happen in bug #831 that could ripple through the current functionality.

ok I'd say this PR completely fixes #93 though which is just adding the missing themes to the picker

**Pull request merged**